ringbuffer/functions.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) The Thanos Community Authors.
 // Licensed under the Apache License 2.0.
 
-package scan
+package ringbuffer
```

Comment:
Moving this to the `ringbuffer` package since the ringbuffer will execute the windowing function in the future. Specialized implementations will do stream based windowing.